db/migrate/20170103124956_add_fk_to_payments.rb

```diff
@@ -0,0 +1,9 @@
+class AddFkToPayments < ActiveRecord::Migration
+  def change
+    add_column :payments, :destination_id, :integer
+    add_column :payments, :source_id, :integer
+
+    add_index :payments, :destination_id
+    add_index :payments, :source_id
+  end
+end
```

Comment:
@NikitaAvvakumov sorry but  I didn't get the note, what changes should be done to `Payment` model ?